sources/north-america/us/ga/ACC_Data.geoJSON

```diff
@@ -0,0 +1,63 @@
+{
+  "type": "Feature",
+  "properties": {
+    "id": "2018_Ortho_Imagery",
+    "name": "2018 Ortho Imagery",
+    "category": "photo",
+    "type": "tms",
+    "url": "https://tiles.arcgis.com/tiles/xSEULKvB31odt3XQ/arcgis/rest/services/2018_Ortho_Imagery/MapServer/tile/{zoom}/{y}/{x}",
+    "country_code": "US",
+    "best": true,
+    "min_zoom": 8,
+    "max_zoom": 20,    	      
+    "attribution": {
+        "url": "https://data-athensclarke.opendata.arcgis.com/",
+        "text": "Athens-Clarke County GIS",
+        "required": true
+    },
+    "valid-georeference": true,
+    "start_date": "2019",
+    "end_date": "2021",
+    "license_url": "https://github.com/osmlab/editor-layer-index/blob/bab9f11b444f0cba6593cd50c25d0cac5aae9fa4/sources/north-america/us/ga/Athens-Clarke_County_Data.pdf",
```

Comment:
I see in the email they said "are free to use for OSM projects" personally I'm not sure if that's sufficient or not.

For CC BY ideally they could complete the CC BY waiver form from https://wiki.osmfoundation.org/wiki/Licence/Waiver_and_Permission_Templates

Reply:
Okay, thanks for the heads up. I'll see if I can add a signed waiver to here.

Reply:
And just for future reference, is it okay if I just leave the link in the geojson as a permalink to the .pdf file?

Reply:
@ianvg The permalink in the pdf is ok. Even better would be to upload the pdf to the osm wiki. This allows being more independent of future GitHub changes.

Reply:
If anyone looks at this pull request- I am still attempting to obtain a letter allowing use of the county’s satellite imagery.

Reply:
[OSM_Imagery_Waiver_signed.pdf](https://github.com/osmlab/editor-layer-index/files/7247127/OSM_Imagery_Waiver_signed.pdf)

Reply:
Dang, I’m getting intimidated by GitHub again. I’ll see if I can figure out how to add this file to my pull request.